server/src/services/domain/user.ts

```diff
@@ -0,0 +1,5 @@
+import {getTelegramUser, addTelegramUser} from '../../../src/bots/telegram/services/storage'
+import { User } from '../../models/user.model';
+
+export const getUser = async (chatId: number): Promise<User> => (await getTelegramUser(chatId) ?? {});
+export const addUser = async (user: User): Promise<Array<User>> => (await addTelegramUser(user) ?? {});
```

Comment:
I am not sure why you add domain wrapper here. My first hesitation would be that it's coupled with Telegram

Reply:
Still is not resolved